src/web/components/Core/Dialog.spec.tsx

```diff
@@ -0,0 +1,37 @@
+import { composeStories } from '@storybook/testing-react';
+import { fireEvent, render, screen } from '@testing-library/react';
+
+import * as stories from './Dialog.stories';
+
+const { Default, WithoutTitle, WithoutCloseButton } = composeStories(stories);
+
+describe('Dialog', () => {
+  it('renders correctly with default props', () => {
+    render(<Default />);
+    const openButton = screen.getByText('Open Dialog');
+    fireEvent.click(openButton);
+
+    expect(screen.getByText('Dialog Title')).toBeInTheDocument();
+    expect(screen.getByText('Dialog content goes here')).toBeInTheDocument();
+    expect(screen.getByText('Close')).toBeInTheDocument();
+  });
+
+  it('renders without title', () => {
+    render(<WithoutTitle />);
+    expect(screen.queryByText('Dialog Title')).not.toBeInTheDocument();
```

Comment:
This isn't really testing anything - there could be a big title with different text! There's no string "Dialog Title" in this story so this wouldn't find anything no matter what.

I think it's fine to just drop this test anyway.